ga4gh/backends.py

```diff
@@ -186,19 +186,19 @@ def searchVariants(self, request):
         else:
             readCols = self._table.columns()[:self._firstSamplePosition]
             callSetIds = request.callSetIds
-            for csid in request.callSetIds:
-                cols = [c for name, c in self._sampleCols[csid]]
+            for callSetId in request.callSetIds:
+                cols = [col for name, col in self._sampleCols[callSetId]]
                 readCols.extend(cols)
         # Now we get the row positions for the sample columns
```

Comment:
All tests pass, ready for merge.
Kerry

On Tue, Jan 6, 2015 at 8:39 PM, Paul Grosu <notifications@github.com> wrote:

> In ga4gh/backends.py
> <https://github.com/ga4gh/server/pull/57#discussion-diff-22566239>:
>
> > @@ -186,19 +186,19 @@ def searchVariants(self, request):
> >          else:
> >              readCols = self._table.columns()[:self._firstSamplePosition]
> >              callSetIds = request.callSetIds
> > -            for csid in request.callSetIds:
> > -                cols = [c for name, c in self._sampleCols[csid]]
> > +            for callSetId in request.callSetIds:
> > +                cols = [col for name, col in self._sampleCols[callSetId]]
> >                  readCols.extend(cols)
> >          # Now we get the row positions for the sample columns
>
> Understandable and agreed :)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ga4gh/server/pull/57/files#r22566239>.
>